components/automate-ui/src/app/ui.component.ts

```diff
@@ -61,6 +62,7 @@ export class UIComponent implements OnInit {
     this.store.select(routeURL).subscribe((url: string) => {
       this.renderNavbar = url.split('/').pop() !== 'add-members';
     });
+    this.store.dispatch(new GetIamVersion());
```

Comment:
This loads the IAM version on app load, as the global project filter only appear if it knows the system is on v2.1.